invoices/sql_migration.go

```diff
@@ -126,3 +130,272 @@ func createInvoiceHashIndex(ctx context.Context, db kvdb.Backend,
 		})
 	}, func() {})
 }
+
+// toInsertMigratedInvoiceParams creates the parameters for inserting a migrated
+// invoice into the SQL database. The parameters are derived from the original
+// invoice insert parameters.
+func toInsertMigratedInvoiceParams(params sqlc.InsertInvoiceParams,
+) sqlc.InsertMigratedInvoiceParams {
```

Comment:
nit: formatting 🤓  (i know there is a war going on re what looks best but current guidelines suggest 

```
func toInsertMigratedInvoiceParams(
	params sqlc.InsertInvoiceParams) sqlc.InsertMigratedInvoiceParams {
```